library/system/src/modules/Kernel.rb

```diff
@@ -169,13 +169,14 @@ def ParseInstallationKernelCmdline
       @cmdline_parsed = true
       return if !(Stage.initial || Stage.cont)
 
-      # live installation does not create /etc/install.inf (bsc#793065)
-      tmp = if Mode.live_installation
+      # Check if /etc/install.inf exists
+      tmp = if SCR.Dir(path(".etc.install_inf")).empty?
```

Comment:
I guess /etc/install.inf should be already copied in the target system when this check is done as well as the read is performed but we have to be careful about it.

Reply:
well, I keep it working as before...so previously it calls SCR.Execute..so I use also SCR for Dir to check existence.